libs/list.liq

```diff
@@ -30,8 +30,11 @@ end
 
 # Get the n-th element of a list (the first element is at position 0), or `default` if element does not exist.
 # @category List
-def rec list.nth(~default, l, n)
-  list.case(l, default, fun (x, l) -> if n == 0 then x else list.nth(default=default, l, n-1) end)
+def list.nth(~default, l, n) =
```

Comment:
So, no recursive `record.member` definition. Doesn't sound too restrictive, though.

Reply:
It could be added though. Again
```
def list.nth(...) = ...
```
is expanded to
```
list = {{ list | nth = (...) -> }}
```
I leave it to you as an exercise to generalize this expansion in the case where we have recursive definitions... :)

Reply:
Yeah I don't think we need to prioritize that just now :-)